extensions/stackdriver/stackdriver.cc

```diff
@@ -70,7 +70,7 @@ int getMonitoringExportInterval() {
   return 60;
 }
 
-// Get logging export interval from node metadata in milliseconds. Returns 60
+// Get logging export interval from node metadata in milliseconds. Returns 10
```

Comment:
What does "returns 10" mean? Is it the default value? That sounds too low for milliseconds.

Reply:
"Returns 10 second".. 10 seconds is the default value..